### DISCUSSION TITLE
Our tests take a long time...  Should we measure them?

### DISCUSSION THREAD

**Discussion by cclauss** [kornia/kornia#1260] [Ideas]
[pytest --durations=10](https://docs.pytest.org/en/6.2.x/usage.html#profiling-test-execution-duration) would give us our ten slowest pytests.  Would it be worthwhile to see if those test can be optimized or dropped?

Also, `pytest -v` generates a lot of text.  Do we need all that?  Does the verbose output slow down our testing?

**Comment by edgarriba** [comment]
@cclauss defenitely - @shijianjian I think that most come from the augmentations. Could you help on that part ?
